clang/lib/Sema/SemaOpenACCClause.cpp

```diff
@@ -639,6 +639,9 @@ OpenACCClause *SemaOpenACCClauseVisitor::VisitVectorLengthClause(
 
 OpenACCClause *SemaOpenACCClauseVisitor::VisitAsyncClause(
     SemaOpenACC::OpenACCParsedClause &Clause) {
+  if (DisallowSinceLastDeviceType<OpenACCAsyncClause>(Clause))
```

Comment:
Can you update the comment on `DisallowSinceLastDeviceType()` to include `async`?